src/mfront.jl

```diff
@@ -0,0 +1,60 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/Materials.jl/blob/master/LICENSE
+
+mbv = MFrontInterface.behaviour
+
+mutable struct MFront <: AbstractMaterial
+    behaviour::BehaviourAllocated
+    behaviour_data::BehaviourDataAllocated
+    offset::Integer
+    equivalent_plastic_strain::AbstractFloat
+
+end
+
+function MFront_Norton()
```

Comment:
The idea is may be here to be able to get the information from the input file.
For example, if the behavior declares material properties, then they shall be retrieved here.
Note that `MFront` has the notion of parameters which are global values for the behavior (and shared by all Gauss points using this behavior). Parameters shall also be initialized from the input file.

Reply:
Same comment as previous. I don't know if I really made a good structure. I just thought I will make this one convenience function to get the example running. Fully agree with your comment.